pset-functions.js

```diff
@@ -135,6 +178,23 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+function scrabbleScore(word) {
```

Comment:
@deedeemedina nice algorithm, however, what's missing is some representation of the letters to their respective score (form the table above). Take a look at the approach used from the solution for more details: https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L178